atak-civ/plugin-examples/plugintemplate/app/src/main/res/layout/preset_timer_cell.xml

```diff
@@ -8,7 +8,7 @@
 
     <TextView
         android:id="@+id/preset_timer_name"
-        android:layout_width="50dp"
+        android:layout_width="125dp"
```

Comment:
https://i.imgur.com/7SfKvIO.png

Reply:
Hmm, I'll take a look at it.

Reply:
Does this happen on both on the preset timer screen and the main screen?

Reply:
@peytonhowell Try it now and see if it's more consistent.

Reply:
@Nboerner Sorry I didn't see your previous message but it is both screens for me but now neither have the problem.
https://i.imgur.com/5gtz4Ag.png - preset
https://i.imgur.com/oauUBio.png - home

It looks better on my side for sure now and the buttons don't overlap anymore. The spacing on the preset screen is considerably skinnier than the home screen though now and I'm not sure if this was intended. It seems like it could be a bit hard to click though using your fingers instead of a mouse, but not sure though. 

I'm not using the correct device anyways (Pixel 2 instead of the galaxy's I think they said they use), so if it's just an issue on this specific device then I don't think it's a big deal.

Reply:
Good catch @peytonhowell , should be patched out now.  If you can, take one more look and make sure everything looks good still on your end.